modules/reference/pages/properties/cluster-properties.adoc

```diff
@@ -5679,7 +5679,11 @@ Always normalize schemas. If set, this overrides the `normalize` parameter in re
 === schema_registry_enable_authorization
 
 Enables ACL-based authorization for Schema Registry requests. When `true`, Schema Registry
-uses ACL-based authorization instead of the default `public/user/superuser` authorization model. Requires authentication to be enabled using the xref:reference:properties/broker-properties.adoc#schema_registry_auth_method[`authentication_method`] property in the `schema_registry_api` broker configuration.
+uses ACL-based authorization instead of the default `public/user/superuser` authorization model. 
+
+ifndef::env-cloud[]
+Requires authentication to be enabled using the xref:reference:properties/broker-properties.adoc#schema_registry_auth_method[`authentication_method`] property in the `schema_registry_api` broker configuration.
```

Comment:
It remains unclear to me how Cloud users should specify the `authentication_method`. @andresaristizabal ?

Reply:
The content has been removed to Cloud version of the topic